appliance_hardening_guide/_topics/db_ssl.md

```diff
@@ -83,21 +91,22 @@ The database appliance now accepts only connections from connecting appliances t
 
 1.  Log in as `root` to the connecting appliance.
 
-2.  Create the `.postgresql` directory in your `root` user home directory.
+3.  Copy the root certificate file from the database to the operating system's list of valid certificates:
 
-        [root@{{ site.data.product.title_short_l }}2 ~]# mkdir /root/.postgresql
+        [root@{{ site.data.product.title_short_l }}2 ~]# DATABASE=[database_appliance_fqdn]
```

Comment:
I think this can be simplfied to just show a terminal prompt and doesn't need the product info...

```suggestion
        $ DATABASE=[database_appliance_fqdn]
```

Reply:
Unless this is meant to contrast between 2 different terminal sessions?  If so, then `root@appliance-host` is just simpler

Reply:
do you want me to make this change across the board?